roofit/xroofit/src/xRooNode.cxx

```diff
@@ -5831,8 +5990,10 @@ TH1 *xRooNode::BuildHistogram(RooAbsLValue *v, bool empty, bool errors, int binS
    RooFitResult *fr = nullptr;
    if (errors) {
       fr = dynamic_cast<RooFitResult *>(fitResult().get()->Clone());
-      if (!fr->_finalPars)
+      if (!GETDMP(fr,_finalPars)) {
```

Comment:
```suggestion
      if (fr->floatParsFinal().empty()) {
```
You can do this after our recent change to always initialize `_finalPars`, right?

Reply:
Yeah but I left it in to be backward compatible. Did we actually make that change to RooFitResult in the end?

Reply:
You are right, we didn't do this in the end! So many changes to RooFit recently, I start to lose the overview

Then I also take back this comment (and the other about `_finalPars`.